--- Vulkan-Loader-sdk-1.3.224.1/loader/vulkan.pc.in.orig	2022-08-15 17:32:06.000000000 +0200
+++ Vulkan-Loader-sdk-1.3.224.1/loader/vulkan.pc.in	2022-11-27 01:05:11.149652405 +0100
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_LIBDIR@
+includedir=@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: @CMAKE_PROJECT_NAME@
 Description: Vulkan Loader
